plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/binance/v4/BinanceSortedGroupV4.java

```diff
@@ -589,8 +593,46 @@ private void createBuySellTxs() {
             quoteRow = ndRow;
         }
 
-        var txsBuySell = new BinanceBeanV4();
-        txsBuySell.setDate(baseRow.getDate());
```

Comment:
I would rather create new field in class BinanceSortedGroupV4 named List<BinanceBeanV4> rowsStakingETH2_0 . Fill the list by the rows and then create new method createBuyFromETH2Staking. The method will contain code below and will be use in method createTransactionsFromMultiRowData. Add also some validation method.

Reply:
pridal jsem validaci a novej list